ios/utils/RCTTouchHandler+RNSUtility.mm

```diff
@@ -0,0 +1,15 @@
+#ifndef RCT_NEW_ARCH_ENABLED
+#import "RCTTouchHandler+RNSUtility.h"
+
+@implementation RCTTouchHandler (RNSUtility)
+
+- (void)rns_cancelTouches
+{
+  [self setEnabled:NO];
+  [self setEnabled:YES];
```

Comment:
Same as above

Reply:
I think changing this code is out of the scope of this PR. I'm reusing already existing & **working** code present in `RNSScreenStack.mm` & this change is only a refactor to improve code readability.